common/proto/common/SDMS.proto

```diff
@@ -86,6 +86,14 @@ enum ProjectRole
     PROJ_ADMIN      = 3;
 }
 
+// Used to specify type of access token contained within message
+enum AccessTokenType {
```

Comment:
This enum needed to be moved to SDMS.proto because defining enums in SDMS_Auth and SDMS_Anon cause issues when datafed-ws.js is importing messages with its current logic